src/main/java/com/ethercamp/harmony/service/BlockchainInfoService.java

```diff
@@ -409,7 +415,9 @@ protected void append(Object eventObject) {
                 if (lastLogs.size() > KEEP_LOG_ENTRIES) {
                     lastLogs.poll();
                 }
-                clientMessageService.sendToTopic("/topic/systemLog", message);
+                if (clientMessageService != null) {
```

Comment:
Too many checks for the same thing.
Let's add a `sendMessage()` method  or use a dummy instance of `clientMessageService`

Reply:
Nice idea! ok